tests/test_init.py

```
@@ -264,6 +601,144 @@ def _create_client(**kwargs: Any) -> Any:
     client.async_close.assert_awaited_once()
 
 
+@pytest.mark.asyncio
+async def test_async_setup_entry_does_not_catch_raw_validation_timeout(
+    monkeypatch: pytest.MonkeyPatch,
+    ph_hass: Any,
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `ph_hass`

(ANN401)